aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt

```diff
@@ -450,7 +450,15 @@ class AztecToolbar : FrameLayout, IAztecToolbar, OnMenuItemClickListener {
         toolbarButtonPlugins.add(buttonPlugin)
 
         val button = findViewById<ToggleButton>(buttonPlugin.action.buttonId)
-        button.setOnClickListener { buttonPlugin.toggle() }
+        val isToolbarAction = ToolbarAction.values().contains(buttonPlugin.action)
+        button.setOnClickListener {
+            if (isToolbarAction) {
+                onToolbarAction(buttonPlugin.action)
```

Comment:
Hi @cameronvoell , thanks for taking another look! Based on the stacktrace it looks like the errors are coming from this change right here. Before these changes, I believe `onToolbarAction` would never have been called from a button click listener.

I will try to find time to look into this deeper today. It's been a while since I was originally working on this and have since moved on to other projects, so I'm not sure how much time I'll be able to dedicate to it now. But I'll do my best